docs/api-contracts/Disputes.md

```diff
@@ -81,24 +81,16 @@ type DisputeResponse = {
 - **Headers:**
     - `Authorization: Bearer <JWT>`
 
-**NOTE:** This endpoint involves files being exchanged. This requires the use of `multipart/form-data` instead of JSON.
-Below is temporary description until this exception is documented
 
-```ts
-interface DisputeCreateRequest {
-  title: string;
-  description: string;
-  evidence: {
-    label: string;
-    file: File;
-  }[];
-  respondent: {
-    full_name: string;
-    email: string;
-    telephone: string;
-  };
+```json
+{
+  "title": "string",
+  "description": "string",
+  "respondent[full_name]": "string",
+  "respondent[email]": "string",
+  "respondent[telephone]": "string",
+  "files": "all the files"
 }
```

Comment:
Please make this a typescript definition for consistency. For files, just denote it as `File`

Reply:
please review change